eng/versioning/external_dependencies.txt

```diff
@@ -397,3 +399,6 @@ storage_com.microsoft.azure:azure-storage;8.4.0
 
 # sdk\appconfiguration\azure-spring-cloud-test-appconfiguration-config\pom.xml
 spring_com.microsoft.azure:azure;1.34.0
+
+# sdk\spring\spring-cloud-azure-service\pom.xml
+spring_redis.clients:jedis;3.8.0
```

Comment:
The jedis version in Spring Boot 3 is 4.3.1 https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/3.0.1/spring-boot-dependencies-3.0.1.pom.